src/backend/integrations/builtin/obs/effects/toggle-obs-source-visibility.ts

```diff
@@ -31,7 +32,30 @@ export const ToggleSourceVisibilityEffectType: Firebot.EffectType<EffectProperti
         categories: ["common"]
     },
     optionsTemplate: `
-<eos-container header="Sources">
+<eos-container ng-show="orphanedSources.length > 0">
+        <div class="effect-info alert alert-warning">
+            <p><b>Warning!</b> 
+                Cannot find {{orphanedSources.length}} sources in this effect. Ensure the correct profile or scene collection is loaded in OBS, and OBS is running.
+            </p>
+        </div>
+</eos-container>
+<setting-container ng-show="orphanedSources.length > 0" header="Missing Sources ({{orphanedSources.length}})" collapsed="true">
+    <div ng-repeat="sceneName in orphanedSources track by $index">
+      <div class="list-item" style="display: flex;border: 2px solid #3e4045;box-shadow: none;border-radius: 8px;padding: 5px 5px;">
+        <div class="pl-5">
+          <span>Scene: {{sceneName.sceneName}},</span>
+            <span ng-if="sceneName.sourceName != null">Name: {{sceneName.sourceName}},</span>
```

Comment:
This will only show for newly added sources, which for existing effects makes it difficult to understand which ones are missing:

![image](https://github.com/user-attachments/assets/7c059187-b8da-48c5-9739-d86a667ba1e1)

Maybe we can find a way to get the source name when collecting the data of the missing sources?

Reply:
that is what we store in the effect we currently only store the scene and the id and action and there is no way to ask obs what a deleted item name is. each item will only show if there it is not sent to us from obs because we didn't store it previously. we cant tell what it is only that it was true false or toggle the scene and the id and not everything is stored that way.

Reply:
sceneName.sourceName ?? "unknown" and dont hide the name field if null? but i feel we still should show the ID

Reply:
![{AF0AFC12-A27B-4F08-8FF9-E0B393B02585}](https://github.com/user-attachments/assets/e2d65c9e-c86d-4fc0-ab6b-158585a3ed77)
but unknown tells us the same as ID not much of anything

Reply:
I think ID unless we have a name is a good way to go about it. Thanks for looking into it!